.gitignore

```diff
@@ -5,3 +5,4 @@ pkg
 _site
 tmp
 *.gem
+.jekyll-cache
```

Comment:
This could have been better handled in a separate issue/PR and discussed separately.
But this fine to include as it supports Jekyll 4 cache.